### PR TITLE
[3/5] Put JIT source inside flatbuffer

### PR DESCRIPTION
**Pull request by qihqi**
Summary: title

Test Plan: unittest

Differential Revision: D34881612

### CODE REVIEW

File: test/cpp/jit/test_flatbuffer.cpp [hunk 1]
```diff
@@ -1135,5 +1135,91 @@ TEST(FlatbufferTest, OperatorTest2) { // NOLINT (use =delete in gtest)
...
+  std::vector<IValue> inputs;
+  inputs.emplace_back(torch::rand({1, 3, 128, 128}));
+  inputs.emplace_back(at::Scalar(2.0));
+  auto ref = m.forward(inputs);
+
+  auto data = save_jit_module_to_bytes(m);
+  Module m2 = jitModuleFromBuffer(data.data());
+  auto res = m2.forward(inputs);
+
+  auto resd = res.toTensor();
```
Comment (iseeyuan) [inline test/cpp/jit/test_flatbuffer.cpp:1165]: Would be nice to add a mobile module from the same data and call forward().

File: test/cpp/jit/test_flatbuffer.cpp [hunk 2]
```diff
@@ -1135,5 +1135,91 @@ TEST(FlatbufferTest, OperatorTest2) { // NOLINT (use =delete in gtest)
   }
 }
 
+Module jitModuleFromBuffer(void* data) {
```
Comment (iseeyuan) [inline test/cpp/jit/test_flatbuffer.cpp:1138]: Would it be parse_and_initialize_jit_module() below?

**Review by iseeyuan — APPROVED** [review]
LGTM

File: test/cpp/jit/test_flatbuffer.cpp (hunk 2 above)
Comment (qihqi) [reply test/cpp/jit/test_flatbuffer.cpp:1138]: yeah though if doing so would need to transfer ownership from flatbuffers::DetachedBuffer to mobile::Module so it's bit annoying.

File: test/cpp/jit/test_flatbuffer.cpp (hunk 1 above)
Comment (qihqi) [reply test/cpp/jit/test_flatbuffer.cpp:1165]: done

**Pull request closed without merging**